### PR TITLE
Create NamishaGoyalPortfolio

### PR DESCRIPTION
**Pull request by namigoel**

### CODE REVIEW

File: Task 2/NamishaGoyalPortfolio [hunk 1]
```diff
@@ -0,0 +1,456 @@
+<!DOCTYPE html>
```
Comment (nimishbongale) [inline Task 2/NamishaGoyalPortfolio:1]: Good work @namigoel!

You may want to do some slight restructuring.

I see that you have a `.html` file. You should add a `.html` extension to your file.

So the final folder structure should be something like:

`Task 2/NamishaGoyalPortfolio/index.html`

**Comment by namigoel** [comment]
I pushed the file, I dont know how to push the folder

**Comment by nimishbongale** [comment]
You have to use something known as a `git client`. 

Find it [here](https://git-scm.com/).

**Comment by namigoel** [comment]
Ok thanks ,I'll see to it :)

**Pull request closed without merging**